lib/lotus/mailer/dsl.rb

```diff
@@ -98,6 +98,122 @@ def templates(value = nil)
         end
       end
 
+      # When a value is given, specify the sender of the email
+      # Otherwise, it returns the sender of the email
+      #
+      # @param value [Object] String or Proc to be evaluated containing the sender of the email
+      #
+      # @return [String] the sender of the email
+      #
+      # @since 0.1.0
+      #
+      # @example With String
+      # class StringMailer
+      #   include Lotus::Mailer
+      #
+      #   from "noreply@example.com"
+      #
+      # end
+      #
+      # @example With Procs
+      # class ProcMailer
+      #   include Lotus::Mailer
+      #
+      #   from = Proc.new { customized_sender }
+      #
+      #   def customized_sender
+      #     "user_sender@example.com"
+      #   end
+      # end
+      def from (value = nil)
+        if value.nil?
+          return new.eval_proc(@from)
+        else
+          @from = value
+        end
+      end
+
+      # When a value is given, specify the email addresses that will be the recipients of the email
+      # Otherwise, it returns the email addresses that will be the recipients of the email
+      #
+      # @param value [Object] String, Array of Strings or Proc to be evaluated containing the email addresses that will be the recipients of the email
+      #
+      # @return [String] the email addresses that will be the recipients of the email
+      #
+      # @since 0.1.0
+      #
+      # @example With String
+      # class StringMailer
+      #   include Lotus::Mailer
+      #
+      #   to "noreply@example.com"
+      #
+      # end
+      #
+      # @example With Array of Strings
+      # class ArrayMailer
+      #   include Lotus::Mailer
+      #
+      #   to ["noreply1@example.com", "noreply2@example.com"]
+      #
+      # end
+      #
+      # @example With Procs
+      # class ProcMailer
+      #   include Lotus::Mailer
+      #
+      #   to = Proc.new { customized_receiver }
+      #
+      #   def customized_receiver
+      #     "user_receiver@example.com"
+      #   end
+      # end
+      def to (value = nil)
+        if value.nil?
+          return new.eval_proc(@to)
+        end
+        if value.is_a?(Array)
```

Comment:
I'm sorry, I didn't understand... We can't just do Array(value).join(',') because value may be a proc, but I don't know if that's what you want :/